src/Common/MessageHandler.ts

```diff
@@ -54,6 +63,10 @@ export function sendMessage(data: any): void {
   });
 }
 
+/**
+ *
+ * @param data Overwrite the data property of the message
```

Comment:
Do we need to add this comment? Doesn't look like the method accepts a param.

Reply:
Looks like the comment was for the previous method. Thanks for the catch!